compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolEnter.java

```diff
@@ -533,6 +535,25 @@ public void visit(BLangImportPackage importPkgNode) {
         this.env.scope.define(pkgAlias, symbol);
     }
 
+    public void initPredeclaredModules(Map<Name, BPackageSymbol> predeclaredModules, SymbolEnv env) {
+        SymbolEnv prevEnv = this.env;
+        this.env = env;
+        for (Name alias : predeclaredModules.keySet()) {
+            ScopeEntry entry = this.env.scope.lookup(alias);
+            if (predeclaredModules.get(alias) != null) {
```

Comment:
This null check is not necessary right? We are iterating through the key set of the predeclared modules map. Therefore, there's no reason for the map to not contain the key it just retrieved

Reply:
Fixed